app/views/foods/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<section id='foods' class="container mx-auto px-4">
+  <div class='mt-5 mb-5 text-end'>
```

Comment:
- [ ] Good job so far but I have an issue with your design. I noticed that your food list doesn't follow the project wireframe. Please fix it in order to make your app design follows the wireframe 👍 

#### Your design
![foods](https://github.com/Solobarine/recipes/assets/6615394/b308498b-0910-4b30-8af4-6f62121dd000)

#### Wireframe
![food_index](https://github.com/Solobarine/recipes/assets/6615394/a371c0c8-ffe5-414d-80ab-dbb30e05f9db)